docs/src/routes/docs/[...3]modules/react.md

```diff
@@ -196,7 +196,10 @@ type UseSetChain = (
 type SetChainOptions = {
   chainId: string
   chainNamespace?: string
-  wallet?: WalletState['label']
+  wallet?: WalletState['label'],
```

Comment:
Might be helpful to include the type notes to this for better understanding of prop usage here